scripts/conflicts/diff.js

```diff
@@ -20,37 +26,90 @@ if (!server || argv.h || argv.help) {
 
 console.log(`Generating diffs for conflicts on ${server}`);
 
+
 const DB = new PouchDB(server);
 
-DB.query('medic-conflicts/conflicts')
+
+const writeToFile = function(path, content) {
+  const writeStream = fs.createWriteStream(path);
+  writeStream.write(JSON.stringify(content, null, 3));
+  writeStream.close();
+};
+
+DB.query('medic-conflicts/conflicts', {reduce:false})
   .then(conflicts => {
     console.log(`Found ${conflicts.rows.length} conflicts`);
 
+    const mainConflictFilesDirectory = './doc-conflicts';
     let ps = Promise.resolve();
+    if (!fs.existsSync(mainConflictFilesDirectory)){
+      try {
+        fs.mkdirSync(mainConflictFilesDirectory);
+      }catch(err){
+        console.log(`Error while creating main conflict folder: ${err}`);
+      }
+    }
     conflicts.rows.forEach(r => {
       ps = ps.then(() => {
-        // console.log(r);
-        if (r.key.length !== 1) {
-          console.log(`${r.id} has ${r.key.length} conflicts, we only want to deal with 1, skipping...`);
-        } else {
-          const docId = r.id;
-          const conflictRev = r.key[0];
-          return DB.get(docId, { open_revs: 'all' })
-            .then(results => {
-              // console.log(results);
-              // Make sure mine and theirs are the right way around
-              const mine = results.find(r => r.ok._rev !== conflictRev).ok;
-              const theirs = results.find(r => r.ok._rev === conflictRev).ok;
-
-              console.log('=====================');
-              console.log(r.id);
-              console.log('=====================');
-              console.log(jsondiff.diffString(mine, theirs));
-              console.log('\n\n');
-            });
-        }
+        const docId = r.id;
+        const arrayRevsIdThatShouldNotBeThere = r.key;
```

Comment:
`arrayRevsIdThatShouldNotBeThere` is pretty unclear to me. is `conflictingRevs` not a suitable name?

Reply:
Replaced with conflictingRevs